chapters/crud/index.md

```diff
@@ -18,7 +18,7 @@ The Flow object now supports a `delete` and `update` method, with a signature si
         f.update(input_fn=random_docs(10))
 
     with f:
-        f.delete(input_fn=random_docs(10))
+        f.delete(input_fn=[d.id for d in random_docs(10)])
```

Comment:
```suggestion
        delete_ids = [d.id for d in random_docs(10)]
        f.delete(delete_ids)
```

Reply:
writing in this way is more clear that we do not need to construct docs for deleting